package WeaponConfig;


message WeaponLvL
{
	required uint32 lvl				= 1;	//
	required uint32 value			= 2;	//属性值
	required uint32 cost_iron		= 3;	//消耗镔铁
	repeated uint32	prob			= 4;	//概率1,2,4,10
}

message WeaponItem
{
	required uint32 	id 				= 1;	//
	required uint32 	unlock_pass		= 2;	//解锁关卡
	required uint32 	pass_cnt		= 3;	//通关次数
	required uint32 	unlock_coin		= 4;	//解锁消耗银币
	required uint32 	unlock_wood		= 5;	//解锁消耗木材
	repeated WeaponLvL 	lvls			= 6;	//对应的每一等级
}

message Weapon
{
	repeated WeaponItem	weapons 	= 1;
}
